src/DataAccess/CarClient.js

```diff
@@ -0,0 +1,6 @@
+
+export class CarClient{
```

Comment:
???? what is this

Reply:
It's handling the fetch functions in order to respect clean architecture instead of the widget itself